content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -76,6 +76,12 @@ Disables all CLI warnings generated by `logger.warn`.
 
 To configure `prisma generate` to [generate a Prisma Client for Data Proxy](/data-platform/classic-projects/data-proxy) on any execution, even when `--data-proxy` is not explicitly set for the CLI command, set the value of this environment value to a truthy value (e.g. `PRISMA_GENERATE_DATAPROXY=true`).
 
+<Admonition>
+
+In Prisma 5.2 and higher, `--accelerate` and `--data-proxy` are no longer required as Prisma Client will determine which engine, if any, is required. To avoid downloading any engine, you can use the [`--no-engines` option](./command-reference) when generating your client.
```

Comment:
It's a bit different: In Prisma 5.2 --accelerate and --data-proxy are not strictly required anymore to use Accelerate or the Data Proxy. If omitted the engines will be included into the generated client out, that's why we now introduced a --no-engine~s~ (singular)

Reply:
Thanks! I updated the wording here to be more specific.